doxygen/mbedtls.doxyfile

```diff
@@ -710,7 +710,7 @@ EXCLUDE_SYMLINKS       = YES
 # against the file with absolute path, so to exclude all test directories
 # for example use the pattern */test/*
 
-EXCLUDE_PATTERNS       =
+EXCLUDE_PATTERNS       = *_internal.h *_wrap.h
```

Comment:
Are we sure we want to remove internal things from the generated documentation that were previously present?